pynestml/codegeneration/spinnaker_code_generator.py

```diff
@@ -216,6 +215,7 @@ def generate_code(self, models: Sequence[ASTModel]) -> None:
         for model in models:
```

Comment:
Can these two for loops be merged into one?

Reply:
Done!